lib/indexers/profiles.js

```diff
@@ -30,6 +30,9 @@ const reset = esClient => {
       return esClient.indices.create({
         index: indexName,
         body: {
+          settings: {
+            'index.max_result_window': 30000
```

Comment:
This may need to be adjusted to match the size of the prod profiles index. It may also not, because realistically whatever is returned for "view all profiles" is unverifiable anyway.